pyscript.core/esm/runtime/_python.js

```diff
@@ -0,0 +1,27 @@
+import { clean, writeFile as writeFileUtil } from "./_utils.js";
+
+// REQUIRES INTEGRATION TEST
+/* c8 ignore start */
+export const run = (runtime, code) => runtime.runPython(clean(code));
+
+export const runAsync = (runtime, code) => runtime.runPythonAsync(clean(code));
+
+export function runEvent(runtime, code, key) {
+    code = `import js;event=js.__events.get(${key});${code}`;
+    return this.run(runtime, code);
+}
+
+const worker = (method) =>
+    function (runtime, code, xworker) {
+        code = `from js import xworker;${code}`;
```

Comment:
Maybe clean up the added variable when you're done?
```suggestion
        code = `from js import xworker;${code}; del xworker`;
```

Reply:
I don't get to control the lifecycle of `xworker` in the *code* so I am not sure that causes undesired side-effects but for one-off events dispatching as in `xworker.onmessage(xworker.event)` this might indeed be a good advice although the message might want to use `xworker` later on ... I need to test, thanks for the idea!

Reply:
`xworker ` is guaranteed to be always the same, it wouldn't be a global worker reference otherwise

Reply:
So why not just set the value once when the interpreter is initialized?

Reply:
mostly 'cause I didn't know this was an option: https://github.com/pyscript/pyscript/pull/1511#discussion_r1224666280

Reply:
P.S. that option is indeed what happens in Lua (wasmoon) and it was my preferred way of doing that within a Worker.